examples/hikari_tanjun_basic_queue/client.py

```diff
@@ -0,0 +1,107 @@
+import os
+import typing
+
+import hikari
+import tanjun
+import lavasnek_rs
+
+
+PREFIX = ","
+TOKEN = os.environ["TOKEN"]
+
+
+class EventHandler:
+    """Handles events from the Lavalink server."""
+
+    async def track_start(self, _: lavasnek_rs.Lavalink, event: lavasnek_rs.TrackStart) -> None:
+        """Handles track start events."""
+        print(f"Track started on guild: {event.guild_id}")
+
+    async def track_finish(self, _: lavasnek_rs.Lavalink, event: lavasnek_rs.TrackFinish) -> None:
+        """Handles track finish events."""
+        print(f"Track finished on guild: {event.guild_id}")
+
+    async def track_exception(
+        self, lavalink: lavasnek_rs.Lavalink, event: lavasnek_rs.TrackException
+    ) -> None:
+        """Handles track exception events."""
+        print(f"Track exception event happened on guild: {event.guild_id}")
+
+        # If a track was unable to be played, skip it
+        skip = await lavalink.skip(event.guild_id)
+        node = await lavalink.get_guild_node(event.guild_id)
+
+        if skip and node:
+            if not node.queue and not node.now_playing:
+                await lavalink.stop(event.guild_id)
+
+
+client = (
+    tanjun.Client.from_gateway_bot(
+        bot := hikari.GatewayBot(token=TOKEN),
```

Comment:
```suggestion
        bot = hikari.GatewayBot(token=TOKEN),
```
Shouldn't be a walrus here.

Reply:
nvm its for running the bot